<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCR_App</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #134074;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; background-color: #eef4ed; padding: 5px; border-radius: 5px; text-align: center;</string>
    </property>
    <property name="text">
     <string>Welcome to OCR Application</string>
    </property>
   </widget>
   <widget class="QLabel" name="description1">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>100</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #eef4ed;</string>
    </property>
    <property name="text">
     <string>OCR System - Extract text from images</string>
    </property>
   </widget>
   <widget class="QLabel" name="description2">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>160</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #eef4ed;</string>
    </property>
    <property name="text">
     <string>OCR Testing - Test the OCR performance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ocr_sys_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8da9c4; color: #eef4ed; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>OCR System</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testing_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>350</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #8da9c4; color: #eef4ed; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>OCR Testing</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
